src/state/atoms/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { WritableAtom, atom } from 'jotai';
+
+// recipe from https://jotai.org/docs/recipes/atom-with-toggle
```

Comment:
@Hyperkid123 thoughts on including this? This allows us to eventually toggle the atom value without the need for checking `isBeta()` in the future.

Reply:
Eventually yes, but now with the URL and local storage being in play its a bit too early for it.